s3cmd

```diff
@@ -2171,6 +2171,33 @@ def cmd_sync(args):
         return cmd_sync_remote2remote(args)
     raise ParameterError("Invalid source/destination: '%s'" % "' '".join(args))
 
+def cmd_getacl(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    uri = S3Uri(args[0])
+    
+    if uri.type != "s3" or not uri.has_bucket():
+        raise ParameterError("Expecting S3 URI instead of '%s'" % args[0])
+    
+    try:
+        acl = s3.get_acl(uri)
+        acl_grant_list = acl.getGrantList()
+        owner = acl.getOwner()
+        output(u"%s (bucket):" % uri.uri())
+        if owner['nick'] is None or owner['nick'] == "":
```

Comment:
You can simplify a little bit by doing something like:
`if not owner['nick']:`
it will cover both cases.